router/connection_maker.go

```diff
@@ -48,7 +48,8 @@ func StartConnectionMaker(ourself *LocalPeer, peers *Peers) *ConnectionMaker {
 		peers:          peers,
 		queryChan:      queryChan,
 		cmdLineAddress: make(map[string]bool),
-		targets:        make(map[string]*Target)}
+		targets:        make(map[string]*Target),
+	}
```

Comment:
Changes, but I disagree. I think `}` at an end-of-line is not a good idea as it produces conflicts for small changes... in fact, I thought the comma in structs was compulsory for this reason...